taxonomies/ingredients.txt

```diff
@@ -25628,12 +25635,23 @@ de:Alaska-Seelachs
 
 fr:crevette
 
+carbon_footprint_fr_foodges_ingredient:fr:Poisson sauvage entier
```

Comment:
Create fr:poisson sauvage entier   and fr:filet de poisson sauvage  maybe, so that you can put those 2 entries